apps/admin/frontend/src/screens/manual_data_import_precinct_screen.tsx

```diff
@@ -401,32 +472,49 @@ export function ManualDataImportPrecinctScreen(): JSX.Element {
         break;
       default: {
         const tally = contestTally.tallies[dataKey];
-        assert(tally);
```

Comment:
The contract of how manual tallies work has changed with this PR.

Previously, we'd expect the contestTallies in a manual tally to include an option tally for every candidate. So when we wanted to add a write-in candidate, we'd add it there. But it's fragile to use that list as the source of truth when there are so many copies of the list (for each precinct).

Now, we expect manual tallies to include an option tally for each official candidate but, for write-in candidates, we can create them on the fly and just assume `undefined` is 0.